web/src/pages/Dashboard.tsx

```diff
@@ -255,7 +255,6 @@ export const NotificationsMap = () => {
           selectionDropdownLeft={isSmallScreen ? undefined : "31.5%"}
           selectionDropdownTop={isSmallScreen ? undefined : "1vw"}
           useRenderer={useNotificationsRenderer}
-          zoom={13}
```

Comment:
any reason in particular to remove this?

Reply:
Yes I wanted to set it to 12 which is default value and no need to pass it separately

The reason is city name is visible on map if not zoomed >12 so set it to 12